pointcloud/src/test/scala/geotrellis/pointcloud/raster/ept/DEMRasterSourceSpec.scala

```diff
@@ -38,9 +38,72 @@ class DEMRasterSourceSpec extends FunSpec with Matchers {
+
+      val res = rs.read()
+      res.nonEmpty shouldBe true
+
+      val tile = res.map(_.tile.band(0)).get
+      tile.dimensions shouldBe Dimensions(128, 128)
+      val (mi, ma) = tile.findMinMaxDouble
+      mi shouldBe 1845.971 +- 1e3
+      ma shouldBe 2028.893 +- 1e3
+    }
+
+    it("should resample RasterSource") {
+      val rs = DEMRasterSource(catalog).resample(100, 100)
+
+      rs.metadata shouldBe EPTMetadata(
+        StringName("src/test/resources/red-rocks/"),
+        CRS.fromEpsgCode(26913),
+        DoubleCellType,
+        new GridExtent(Extent(481968.0, 4390186.0, 482856.0, 4391074.0), 6.9375, 6.9375, 128, 128),
+        List(CellSize(6.9375,6.9375), CellSize(3.46875,3.46875), CellSize(1.734375,1.734375), CellSize(0.8671875,0.8671875), CellSize(0.43359375,0.43359375)),
+        Map("points" -> "4004326", "pointsInLevels" -> "15366,186189,465711,2297397,1039663", "minz" -> "1843.0", "maxz" -> "2030.0")
+      )
+
+      rs.gridExtent shouldBe new GridExtent(Extent(481968.0, 4390186.0, 482856.0, 4391074.0),8.88, 8.88,100, 100)
+      rs.crs shouldBe CRS.fromEpsgCode(26913)
+
+      val res = rs.read()
+      res.nonEmpty shouldBe true
+
+      val tile = res.map(_.tile.band(0)).get
+      tile.dimensions shouldBe Dimensions(100, 100)
+      val (mi, ma) = tile.findMinMaxDouble
+      mi shouldBe 1846.696 +- 1e3
+      ma shouldBe 2027.657 +- 1e3
+    }
+
+    it("should reproject RasterSource") {
+      val rs = DEMRasterSource(catalog).reproject(LatLng)
+
+      rs.metadata shouldBe EPTMetadata(
+        StringName("src/test/resources/red-rocks/"),
+        CRS.fromEpsgCode(26913),
+        DoubleCellType,
+        new GridExtent(Extent(481968.0, 4390186.0, 482856.0, 4391074.0), 6.9375, 6.9375, 128, 128),
+        List(CellSize(6.9375,6.9375), CellSize(3.46875,3.46875), CellSize(1.734375,1.734375), CellSize(0.8671875,0.8671875), CellSize(0.43359375,0.43359375)),
+        Map("points" -> "4004326", "pointsInLevels" -> "15366,186189,465711,2297397,1039663", "minz" -> "1843.0", "maxz" -> "2030.0")
+      )
+
+      rs.gridExtent shouldBe new GridExtent(Extent(-105.21023644880934, 39.661268543413485, -105.19987676348154, 39.669309977479124), 7.244535194267097E-5,7.244535194267097E-5, 143, 111)
+      rs.crs shouldBe LatLng
       val res = rs.read()
       res.nonEmpty shouldBe true
-      res.map(_.tile.band(0).findMinMaxDouble) shouldBe Some(1845.9715706168827 -> 2028.8939339826734)
+
+      val tile = res.map(_.tile.band(0)).get
+      tile.dimensions shouldBe Dimensions(143, 111)
+      val (mi, ma) = tile.findMinMaxDouble
+      mi shouldBe 1845.683 +- 1e3
+      ma shouldBe 2028.15 +- 1e3
+    }
+
+    ignore("rasterizer bug") {
+      val ge = new GridExtent[Long](Extent(481968.0, 4390186.0, 482718.32558139536, 4390537.069767442), 6.883720930232645, 6.883720930227462, 109, 51)
+      val rs = DEMRasterSource(catalog).resampleToRegion(ge)
+      GeoTiff(rs.read().get, rs.crs).write("/tmp/test.tiff")
```

Comment:
@jpolchlo I could catch a triangulation stripe!

Reply:
This issue is outside of this scope https://github.com/geotrellis/geotrellis-pointcloud/issues/47